### PR TITLE
Mobile Apps Audit 2022

### PR DESCRIPTION
**Pull request by ChristieBavelaar**
###Mattermost Developer Documentation Content Audit 2022

#### Summary
Update of Mobile Apps documentation index page

### CODE REVIEW

**Review by enahum — APPROVED** [review]

**Comment by ChristieBavelaar** [comment]
@cwarnermm I updated the branch

**Comment by cwarnermm** [comment]
@ChristieBavelaar - Unfortunately, I'm seeing a "branch is out of date" message on this PR, and I'm unable to update it myself because it's a fork. Once you've updated your copy of master, then please merge master into your branch for this PR. Thanks so much!

File: site/content/contribute/mobile/_index.md [hunk 1]
```diff
@@ -2,20 +2,31 @@
...
+date: "2022-03-15T12:01:23-04:00"
 weight: 4
 ---
 
 The Mattermost mobile apps are written in [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript) using [React Native](https://facebook.github.io/react-native/).
 
 ## Repository
 
-https://github.com/mattermost/mattermost-mobile
+Further explanation of the file structure can be found in [the Developer Setup](https://developers.mattermost.com/contribute/mobile/developer-setup/structure/)
```
Comment (cwarnermm) [inline site/content/contribute/mobile/_index.md:13]:
```suggestion
Further explanation of the file structure can be found in [the Developer Setup](https://developers.mattermost.com/contribute/mobile/developer-setup/structure/).
```

File: site/content/contribute/mobile/_index.md [hunk 2]
```diff
@@ -2,20 +2,31 @@
...
 
-## Channel
 
-https://community.mattermost.com/core/channels/native-mobile-apps
+## Mobile App Contributor Resources
+ - [GitHub Repository](https://github.com/mattermost/mattermost-mobile) - Get the code, report issues, or submit PRs.
+ - [Running the app](/contribute/mobile/developer-setup/run/) - compile recommendations
```
Comment (cwarnermm) [inline site/content/contribute/mobile/_index.md:18]:
```suggestion
 - [Running the app](/contribute/mobile/developer-setup/run/) - Compile recommendations.
```

File: site/content/contribute/mobile/_index.md [hunk 3]
```diff
@@ -2,20 +2,31 @@
...
+ - [Channel](https://community.mattermost.com/core/channels/native-mobile-apps)  - Use the channel to interact with other Mattermost Mobile developers.
+ - [Help Wanted](https://mattermost.com/pl/help-wanted-mattermost-mobile) - Find help wanted tickets here
```
Comment (cwarnermm) [inline site/content/contribute/mobile/_index.md:20]:
```suggestion
 - [Help Wanted](https://mattermost.com/pl/help-wanted-mattermost-mobile) - Find help wanted tickets here.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you for your patience, @ChristieBavelaar. While I haven't been able to get the generated preview working in the PR itself, I was able to pull your PR down locally and build it to see your great updates. I've provided feedback inline.

**Comment by ChristieBavelaar** [comment]
@cwarnermm Thank you for your suggestions, I have commited them.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**